macro/Version.py

```diff
@@ -36,19 +36,15 @@
 Dependencies = []
 
 # configure the active set of distros
-from macroutils import distro_names as distros
+from macroutils import distro_names, distro_names_hidden
 
-if 'boxturtle' in distros:
-    distros.remove('boxturtle')
-if 'unstable' in distros:
-    distros.remove('unstable')
+distros = [d for d in distro_names if d not in distro_names_hidden]
```

Comment:
The commit where you moved the hidden distros in a single location is good as is. The package and stack headers now use the same list of non-hidden distros.

This case is different for the version macro. It is being used within the content of wiki pages and marks e.g. a paragraph describing a certain feature with the ROS distro when it was added. So it needs to continue to work for all distros. I don't know why this should exclude "boxturtle" and "unstable" (but I wouldn't touch this).